plugins/googleanalytics/test-src/org/pentaho/di/trans/steps/googleanalytics/GoogleAnalyticsApiFacadeTest.java

```diff
@@ -0,0 +1,19 @@
+package org.pentaho.di.trans.steps.googleanalytics;
```

Comment:
Thank you so much for adding a test! 
Please NOTE that tests also require the proper license headers at the top of each file.